checkpoint/side_milestone_handler.go

```diff
@@ -22,12 +22,6 @@ func SideHandleMsgMilestone(ctx sdk.Context, k Keeper, msg types.MsgMilestone, c
 	// logger
 	logger := k.Logger(ctx)
 
-	//Check whether the chain has reached the hard fork length
-	if ctx.BlockHeight() < helper.GetMilestoneHardForkHeight() {
```

Comment:
why have we removed this?

Reply:
This is redundant, Msg will never reach this if the chain has not crossed the hardfork.